README.md

```diff
@@ -0,0 +1,256 @@
+
+# Capdrupal version
+
+Capdrupal Gem Version | Branch | Capistrano Version | Drupal Version
+--------------------- | ------ | ------------------ | --------------
+7.x                   | d7     | 2                  | 7.x
+8.x                   | d8     | 3.11               | 8.x
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'capdrupal', '~>8.0'
+```
+
+And then execute:
+
+```shell
+bundle
+```
+
+Or install it yourself if [gems](http://rubygems.org) is installed on your system:
+
+```shell
+gem install capdrupal
+```
+
+## Configuration
+
+First, go to your project directory and launch Capistrano.
+
+```shell
+cd path/to/your/drupal/project/
+cap install
+```
+
+Capistrano will create the following skeleton
+
+```
+.
+├── Capfile
+├── config
+│   └── deploy.rb
+│   └── deploy
+│       └── production.rb
+│       └── staging.rb
+├── lib
+│   └── capistrano
+│        └── tasks
+
+```
+
+Create two files `Capfile` and `config/deploy.rb`. Open `Capfile` and set the dependencies.
+
+```ruby
+# Load DSL and set up stages.
+require 'capistrano/setup'
+
+# Include default deployment tasks.
+require 'capistrano/deploy'
+
+# Composer is needed to install drush on the server.
+require 'capistrano/composer'
+
+# Drupal Tasks.
+require 'capdrupal/capistrano'
+
+# Load custom tasks from `lib/capistrano/tasks` if you have any defined.
+Dir.glob('config/capistrano/tasks/*.rake').each { |r| import r }
+```
+
+Then, go to `config/deploy.rb` to set the parameters of your project. First you have to define the general information about the user, server and the app himself.
+
+```ruby
+set :application, 'application-name'
+set :repo_url, 'git@github.com:company/application.git'
+
+server 'ssh.example.org', user: 'username', roles: %w{app db web}
+```
+
+The specific Drupal information
+
+```ruby
+set :install_composer, true
+set :install_drush, true
+
+set :app_path, 'web'
+set :config_path, 'config/sync'
+
+# Setup the backup before/after failed strategy.
+set :backup_path, 'backups'
+set :keep_backups, 5
+
+# Link file settings.php
+set :linked_files, fetch(:linked_files, []).push("#{fetch(:app_path)}/sites/default/settings.php", "drush/drush.yml")
+
+# Link dirs files and private-files
+set :linked_dirs, fetch(:linked_dirs, []).push("#{fetch(:app_path)}/sites/default/files")
+```
+
+Then, all the others information related to your Git repository or debug level
+
+```ruby
+# Default value for :scm is :git
+set :scm, :git
+
+# Default value for :log_level is :debug
+set :log_level, :debug
+```
+
+Finally, set the deployment to use the proper Drupal 8 strategy
+
+```ruby
+namespace :deploy do
+  # Ensure everything is ready to deploy.
+  after "deploy:check:directories", "drupal:db:backup:check"
+
+  # Backup the database before starting a deployment and rollback on fail.
+  # before :starting, "drupal:db:backup"
```

Comment:
```suggestion
  # after :updated, "drupal:db:backup"
```

This change allow to use the rush version in vendors, and also to have a fresh backup.